jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -120,6 +124,12 @@ public Response registerInstallation(
         // otherwise we register a new installation:
         logger.finest("Mobile Application on device was launched");
 
+        //let's do update the analytics
+
+        String pushIdentifier = extractPushIdentifier(request);
+        if(pushIdentifier != null) {
```

Comment:
perhaps also update the endpoint doc?

On Monday, April 13, 2015, Sebastien Blanc <notifications@github.com> wrote:

> In
> jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523#discussion_r28223212>
> :
>
> > @@ -120,6 +124,12 @@ public Response registerInstallation(
> >          // otherwise we register a new installation:
> >          logger.finest("Mobile Application on device was launched");
> >
> > +        //let's do update the analytics
> > +
> > +        String pushIdentifier = extractPushIdentifier(request);
> > +        if(pushIdentifier != null) {
>
> push identifier is the key concept of this PR ;) but let me document the
> extratPushIdentifier method.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523/files#r28223212>
> .
>


-- 
Sent from Gmail Mobile